src/shared/components/case-editor/case-edit-page/case-edit-page.scss

```diff
@@ -3,3 +3,9 @@
   border-right-color: #ffcc02;
   border-right-style: solid;
 }
+
+.validationError {
```

Comment:
_Nitpick:_
As mentioned in the .html file, this would be better named `validation-error`.

Reply:
updated